src-self-hosted/link/MachO.zig

```diff
@@ -15,6 +17,14 @@ pub const base_tag: Tag = File.Tag.macho;
 
 base: File,
 
+/// Stored in native-endian format, depending on target endianness needs to be bswapped on read/write.
+/// Same order as in the file.
+segment_cmds: std.ArrayListUnmanaged(macho.segment_command_64) = std.ArrayListUnmanaged(macho.segment_command_64){},
+
+/// Stored in native-endian format, depending on target endianness needs to be bswapped on read/write.
+/// Same order as in the file.
+sections: std.ArrayListUnmanaged(macho.@"section_64") = std.ArrayListUnmanaged(macho.@"section_64"){},
```

Comment:
Oh oops, I was looking at `macho.@"section"` and somehow my brain automatically simply expanded the name to `@"section_64"` instead of `section_64`. Thanks for spotting that one!